sanity/keylime-secure_mount/main.fmf

```diff
@@ -0,0 +1,21 @@
+summary: Tests secure mount on keylime agent
+description: |
+ Running keylime verifier and registrar on localhost.
+ Start agent and check creating and mount secure dir.
+ Manually mount secure dir with wrong FS and restart
+ agent. Check that agent fail.
+contact: Patrik Koncity <pkoncity@redhat.com>
+component:
+  - keylime
+test: ./test.sh
+framework: beakerlib
+tag:
+  - CI-Tier-2
+recommend:
+  - keylime
+adjust:
+  - when: agent == python
```

Comment:
Is this necessary? I think we are not going to backport test to old Fedora releases which are the only releases that have the Python agent. Btw, I believe that Python agent does such mount too.

Reply:
I can remove it, but I need to handle part where remove secure mount setup from unit file. Secure mount setup in unit file is just for rust agent.